sdk/pom.xml

```diff
@@ -57,5 +57,11 @@
             <version>5.0.0-alpha.14</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
we are using [junit 5](https://github.com/opentdf/java-sdk/blob/cab573ad79d21f27e06ef244819d1001cd469a10/sdk/pom.xml#L20-L23) so I don't think that we need this import.